RevenueCat/Scripts/Purchases.cs

```diff
@@ -411,8 +396,41 @@ public void RestorePurchases(CustomerInfoFunc callback)
     [Obsolete("Deprecated, use set<NetworkId> methods instead.", true)]
     public void AddAttributionData(string dataJson, AttributionNetwork network, string networkUserId = null) { }
 
+    /// <summary>
+    /// Callback function for <see cref="Purchases.LogIn"/>. 
+    /// </summary>
+    /// <param name="customerInfo"> The <see cref="CustomerInfo"/> of the user if the request was successful.
+    /// Null otherwise.</param>
+    /// <param name="created"> True if the user was registered in RevenueCat's database for the first time upon
+    /// making this call. False if a user with this <c>appUserID</c> already existed in RevenueCat's database for
+    /// this app.</param>
+    /// <param name="error"> If the request was unsuccessful, contains an error. Null otherwise. </param>
+    public delegate void LogInFunc(CustomerInfo customerInfo, bool created, Error error);
```

Comment:
This was the same as the previous name, but I wonder if this would be better named `LogInDelegate` or `LogInCallback` (Not familiar with Unity or C# so this might be the style here). Not for this PR in any case.

Reply:
The Unity SDK needs to have a _type_ for the callback, and it also needs to have a variable that stores a callback that was passed in, to call it once the method is finished. 

Our naming scheme right now is: 
The _type_ of callback for the method is defined as `FooFunc`, and then the variable is `FooCallback`. 

Full disclosure: I've no idea why we follow this naming scheme. And / or whether that's even remotely standard.

Reply:
I'll ask our more knowledgeable C# devs